spec/controllers/announcements_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe AnnouncementsController, type: :controller do
+  render_views
+
+  let(:stub_user) { create :user, :coronovirus_editor, name: "Name Surname" }
+  let(:coronavirus_page) { create :coronavirus_page, :of_known_type }
+  let(:slug) { coronavirus_page.slug }
+  let(:announcement) { create :announcement, coronavirus_page: coronavirus_page }
+  let!(:live_stream) { create :live_stream, :without_validations }
```

Comment:
We need this for the stubbing of the coronavirus page, we started to look into this but it seems intertwined with lots of other bits of code. Maybe we could make a separate ticket to sort this out?

Reply:
What error are you getting if you remove this value?

Reply:
Hi Leena, sorry didn't fix this due to time. I think the issue is because when the post happens, the draft updater calls the `ContentBuilder` which sets up a `live_stream` https://github.com/alphagov/collections-publisher/blob/master/app/services/coronavirus_pages/content_builder.rb#L14.

`live_stream = LiveStream.last` https://github.com/alphagov/collections-publisher/blob/master/app/services/coronavirus_pages/content_builder.rb#L89 expects that a LiveStream instance exists. I guess the solution will be to add some sort of check in the `persisted_live_stream_data` for whether an instance exists or not... I am not sure if it is always needed. Possibly out of scope for this but something we could consider investigating separately?